scripts/deployChainlinkPricer.js

```diff
@@ -1,30 +1,33 @@
-const yargs = require("yargs");
+const yargs = require('yargs')
 
-const ChainlinkPricer = artifacts.require("ChainlinkPricer.sol");
+const ChainlinkPricer = artifacts.require('ChainlinkPricer.sol')
 
-module.exports = async function(callback) {
-    try {
-        const options = yargs
-            .usage("Usage: --network <network> --bot <bot> --asset <asset> --aggregator <aggregator> --oracle <oracle> --gas <gasPrice>")
-            .option("network", { describe: "Network name", type: "string", demandOption: true })
-            .option("bot", { describe: "Bot address", type: "string", demandOption: true })
-            .option("asset", { describe: "Asset address", type: "string", demandOption: true })
-            .option("aggregator", { describe: "Chainlink aggregator address", type: "string", demandOption: true })
-            .option("oracle", { describe: "Oracle module address", type: "string", demandOption: true })
-            .option("gas", { describe: "Gas price in WEI", type: "string", demandOption: false })
-            .argv;
+module.exports = async function(callback, k) {
+  try {
+    const options = yargs
+      .usage(
+        'Usage: --network <network> --bot <bot> --asset <asset> --aggregator <aggregator> --oracle <oracle> --gas <gasPrice>',
+      )
+      .option('network', {describe: 'Network name', type: 'string', demandOption: true})
+      .option('bot', {describe: 'Bot address', type: 'string', demandOption: true})
+      .option('asset', {describe: 'Asset address', type: 'string', demandOption: true})
+      .option('aggregator', {describe: 'Chainlink aggregator address', type: 'string', demandOption: true})
+      .option('oracle', {describe: 'Oracle module address', type: 'string', demandOption: true})
+      .option('gas', {describe: 'Gas price in WEI', type: 'string', demandOption: false}).argv
 
-        console.log(`Deploying chainlink pricer contract on ${options.network} 🍕`)
+    console.log(`Deploying chainlink pricer contract on ${options.network} 🍕`)
 
-        const tx = await ChainlinkPricer.new(options.bot, options.asset, options.aggregator, options.oracle, {gasPrice: options.gas});
+    const tx = await ChainlinkPricer.new(options.bot, options.asset, options.aggregator, options.oracle, {
+      gasPrice: options.gas,
+      gas: 470000,
```

Comment:
not sure we should make this static, maybe it add another argument similar to `options.gas` @antoncoding

Reply:
let me actually update old `gas` arg to `gasPrice`, and make this one `gasLimit` to avoid confusion.